### PR TITLE
Fix mobile nav horizontal overflow issue

### LINKED ISSUE #4586: it looks like we have a horizontal scrollbar at medium and smaller breakpoints
Chrome:

image1

Firefox:

image

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #4586

The mobile nav section (black background section below) was having overflow issue

image

### CODE REVIEW

File: source/js/buyers-guide/components/primary-nav/primary-nav.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@
   height: 100vh;
   top: 0;
   left: 0;
-  width: 100vw;
+  right: 0;
```
Comment (mmmavis) [inline source/js/buyers-guide/components/primary-nav/primary-nav.scss:62]: `left: 0;` with `right: 0;` to make an absolute positioned element go full width

**Comment by Pomax** [comment]
Looks like this still happens for s/xs (but not for m anymore)...

Chrome:

image

Firefox:

image

**Comment by mmmavis** [comment]
Alright! I'm gonna use this as an opporutunity to clean up `primary-nav.scss` code for both Foundation site and PNI. They seem to be sharing quite a bit of code.

**Comment by mmmavis** [comment]
Closing as it requires work to fix our existing `.full-bleed` related rules which is annoying to fix and I don't have the bandwidth for it this sprint.

**Pull request closed without merging**